docs/fields.rst

```diff
@@ -48,15 +48,14 @@ The decorator ``marshal_with`` is what actually takes your data object and appli
 field filtering.  The marshalling can work on single objects, dicts, or
 lists of objects.
 
-Note: marshal_with is a convenience decorator, that is functionally equivalent to the following ``return marshal(db_get_todo(), resource_fields), 200``.
-This explicit expression can be used to return other HTTP status codes than 200 along with a successful response (see ``abort`` for errors).
+Note: marshal_with is a convenience decorator, that is functionally equivalent to ``return marshal(db_get_todo(), resource_fields), 200``.
```

Comment:
The code example here is probably best formatted as a separate code block instead of inline with the paragraph.